packages/babel-plugin-transform-decorators-2/test/fixtures/examples/spare/exec.js

```diff
@@ -0,0 +1,30 @@
+// makes an undecorated copy of the method
+function spare(name, isStatic) {
+  return function(descriptor) {
+    const clone = {
+      kind: "property",
+      isStatic: !!isStatic,
+      key: name,
```

Comment:
Currently what I've interpreted from the spec is that it should "merge". By that I mean it will just take on the value of the last property added with the key "foo". See [decorateElement](https://tc39.github.io/proposal-decorators/#sec-decorate-element) step 6. Also see `mergeDuplicateElements` helper